app/workers/commit_monitor_handlers/commit_range/rubocop_checker.rb

```diff
@@ -57,17 +57,9 @@ def rubocop_comments
     MessageBuilder.new(results, branch).comments
   end
 
-  def write_to_github
-    logger.info("Updating PR #{pr_number} with rubocop comment.")
-
-    branch.repo.with_github_service do |github|
-      @github = github
-      replace_rubocop_comments
-    end
-  end
```

Comment:
@chrisarcand Also should be an attr_reader you can kill here as well (you can do in sep PR)